src/main/java/org/broadinstitute/hellbender/utils/runtime/RuntimeUtils.java

```diff
@@ -50,4 +50,18 @@ public static String toolDisplayName(final Class<?> toolClass) {
                 toolClass.getSimpleName();
     }
 
+    /**
+     * Get the version of GATK.
+     * @return A string representing the version of the GATK software, or the string "Unavailable" if
+     * the version can't be determined. Usually, the version String is retrieved from the manifest in
+     * the GATK jar. However, when running tests through Gradle/IntelliJ, there is no uber jar, only
+     * the compiled .class files. In that case, the string "Unavailable" is returned.
+     */
+    public static String getGATKVersionString() {
+        final String versionString = RuntimeUtils.class.getPackage().getImplementationVersion();
```

Comment:
This won't work on downstream toolkits if they bundle GATK tools - it will pick the version of it's uber-jar and thus it will always fail validation for python executor.

Reply:
Yes, I'm not really satisfied with the versioning scheme we have now either (and perhaps this method is misnamed). For the Python executor, all that matters is that the version returned by this method matches the version of the `gatktool` python package being run. With this PR, GATK ensures this by stamping both of those with the (same GATK version value) at build-time.

If a downstream toolkit wants to use Python in a tool via the executor, it will have to use a similar system (create a conda, or equivalent, environment of it's own, and make sure that the GATK `gatktool` python package running in it is stamped with the same value returned here.

Reply:
Because this method is static, it cannot be overriden and there is no way to pass a different version to the executor (see my other comment). In addition, the `getGATKVersionString` is also used in the CLP class, which will grab the manifest information and it is inaccurate for downstream toolkits because it will get their manifest version.